plugins/common/http/config.go

```diff
@@ -52,7 +53,14 @@ func (h *HTTPClientConfig) CreateClient(ctx context.Context, log telegraf.Logger
 		Timeout:   time.Duration(timeout),
 	}
 
-	client = h.OAuth2Config.CreateOauth2Client(ctx, client)
+	h.OAuth2Config.TokenURL = fmt.Sprintf("%v", ctx.Value("url"))
```

Comment:
Is this `Sprintf` needed?

Reply:
No longer using this line of code and `Sprintf`